src/index.ts

```diff
@@ -1,22 +1,48 @@
 import axios from 'axios';
+import Ain from '@ainblockchain/ain-js';
 import Assets from './assets';
-import { ENDPOINT } from './constants';
+import Discord from './discord';
+import { AINFT_SERVER_ENDPOINT, BLOCKCHAIN_CHAINID, BLOCKCHAIN_ENDPOINT } from './constants';
+import { Account } from './types';
+
 export default class AinftJs {
   private baseUrl: string;
+  public accessAccount: Account;
+  public signature: string;
+  public signatureData: any;
   public assets: Assets;
+  public discord: Discord;
+  public ain: Ain;
 
-  constructor(baseUrl = ENDPOINT) {
+  constructor(baseUrl = AINFT_SERVER_ENDPOINT, accessAccount: Account) {
```

Comment:
How can we get accessAccount? By env?

Reply:
we will give accessAccount in offline (after we made it.). 
The accessAccount will be created per app. User that want to create app and project in ainft factory contacts us and can get accessAccount.

and we have to more discussion how to give accessAccount  to user automatically. (after Q3?)